src/components/PreviousButton/index.tsx

```diff
@@ -1,11 +1,11 @@
 // SPDX-License-Identifier: MPL-2.0
 // Copyright (C) 2020 - 2021 Gemeente Amsterdam
-import PropTypes from 'prop-types'
 import styled from 'styled-components'
 import { ChevronLeft } from '@amsterdam/asc-assets'
 import { themeColor } from '@amsterdam/asc-ui'
 
 import Button from 'components/Button'
+import type { FunctionComponent } from 'react'
```

Comment:
please move this line to the top, ordering stuff

Reply:
Your remarks has been resolved (as in: done and done)